index.js

```diff
@@ -89,11 +117,6 @@ function getOptions(uri, o, method) {
         o.timeout = 2 * 60 * 1000; // 2 minutes
     }
 
-    // Default pool options: Don't limit the number of sockets
-    if (!o.pool) {
-        o.pool = {maxSockets: Infinity};
-    }
-
```

Comment:
@arlolra oh indeed, sorry to ping you while I could find the answer on my own... We don't use `globalAgent` any more, but instead use `request` `agentClass` feature, so this is not needed any more. I'm closing the PR as all the rest was done in #21